components/icons/NettestGroupCircumvention.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Icon from 'react-icon-base'
+
+const NettestGroupCircumvention = props => (
```

Comment:
This icon should be generated using the npm command `create-icons`: https://github.com/ooni/design-system/blob/master/package.json#L59.

The svg of the icon should instead live inside of https://github.com/ooni/design-system/tree/master/components/svgs/icons

Reply:
Generated with the script now.